backend/src/test/java/com/woowacourse/levellog/application/PreQuestionServiceTest.java

```diff
@@ -0,0 +1,419 @@
+package com.woowacourse.levellog.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import com.woowacourse.levellog.common.exception.InvalidFieldException;
+import com.woowacourse.levellog.common.exception.UnauthorizedException;
+import com.woowacourse.levellog.levellog.domain.Levellog;
+import com.woowacourse.levellog.member.domain.Member;
+import com.woowacourse.levellog.prequestion.domain.PreQuestion;
+import com.woowacourse.levellog.prequestion.dto.PreQuestionDto;
+import com.woowacourse.levellog.prequestion.exception.PreQuestionNotFoundException;
+import com.woowacourse.levellog.team.domain.Participant;
+import com.woowacourse.levellog.team.domain.Team;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("PreQuestionService의")
+public class PreQuestionServiceTest extends ServiceTest {
+
+    @Nested
+    @DisplayName("save 메서드는")
+    class Save {
+
+        @Test
+        @DisplayName("사전 질문을 생성한다.")
+        void save() {
+            //given
+            final String preQuestion = "로마가 쓴 사전 질문";
+            final PreQuestionDto preQuestionDto = new PreQuestionDto(preQuestion);
+
+            final Team team = teamRepository.save(
+                    new Team("선릉 네오조", "목성방", LocalDateTime.now().plusDays(3), "네오조.img", 1));
+            final Member author = memberRepository.save(new Member("알린", 12345678, "알린.img"));
+            final Member questioner = memberRepository.save(new Member("로마", 56781234, "로마.img"));
```

Comment:
```suggestion
            final Team team = saveAndGetTeam("선릉 네오조", 1);
            final Member author = saveAndGetMember("알린");
            final Member questioner = saveAndGetMember("로마");
```
이런 식으로 메서드 추출을 하면 테스트 가독성이 더 올라간다고 생각해요. 알린은 어떠신가요? 🤔

Reply:
오 좋아요 수정했습니다